src/overlays/actors/ovl_Obj_Tokei_Tobira/z_obj_tokei_tobira.c

```diff
@@ -36,16 +37,150 @@ static InitChainEntry D_80ABD750[] = {
     ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
 };
 
-#endif
+Vec3f D_80ABD760 = { 0.0f, 0.0f, 80.0f };
+s16 D_80ABD76C[2] = { 0xC000, 0x4000 };
+s32 D_80ABD770[2] = { 0x060012B0, 0x06001590 };
```

Comment:
Each element of this array is passed to `DynaPolyActor_LoadMesh`. This function expects a `CollisionHeader *`, which means this  array is actually an array of pointers. Please fix the type.
This are also segmented pointers (you can tell by looking at the initial `0x06`). Please use the corresponding symbols instead

Reply:
Didn't quiet understand this one